pages.zh/common/javac.md

```diff
@@ -1,6 +1,7 @@
 # javac
 
 > Java 程序编译器。
+> 更多信息：<https://docs.oracle.com/en/java/javase/13/docs/specs/man/javac.html>.
```

Comment:
Maybe we should link to the Java 17 docs instead?

Java 17 is the latest LTS version of Java (as of 14th September) and Java 11 before that.
I think it'd more useful than linking to an LTS release which is no longer maintained.

More info: https://en.wikipedia.org/wiki/Java_version_history

```suggestion
> 更多信息：<https://docs.oracle.com/en/java/javase/17/docs/specs/man/javac.html>.
```